Source/WTF/wtf/Vector.h

```diff
@@ -276,16 +281,31 @@ struct VectorTypeOperations
         VectorMover<VectorTraits<T>::canMoveWithMemcpy, T>::move(src, srcEnd, dst);
     }
 
+    static void move(std::span<T> src, std::span<T> dst)
+    {
+        VectorMover<VectorTraits<T>::canMoveWithMemcpy, T>::move(std::to_address(src.begin()), std::to_address(src.end()), std::to_address(dst.begin()));
```

Comment:
This doesn't check the size of `dst`.

Reply:
Yes, Ideally it would but this is not a regression in this PR. I still think this is a step in the right direction. I'm trying to reduce the risk of introducing a perf regression in this PR by keeping the change minimal.

That said, A/B testing has been down for a few days so I haven't been able to do any perf testing for this patch yet.